src/config/project_settings.js

```diff
@@ -12,7 +12,7 @@ module.exports = {
   // Old epic bug blocking <2019 projects/features
   LEGACY_EPIC_BUG_NUMBER: "1433315",
   // What component should new bugs be filed in?
-  FILE_NEW_BUGZILLA_COMPONENT: "Activity Streams: Newtab",
+  // FILE_NEW_BUGZILLA_COMPONENT: "Activity Streams: Newtab",
```

Comment:
Can you delete this instead of commenting it out?

Reply:
yep!

Reply:
Looks like this was deleted in https://github.com/k88hudson/bugzy/commit/33f726b1e68d0e13d104a41d1bd8fa778d3b65c6 but got restored from a force push and was committed:

https://github.com/k88hudson/bugzy/commit/4fa4b08aac093e453f92c4e4a466ac1b6399be7f#diff-75d6aa8e05b9123d4745966a7271dfd4L15